File: Progamming Languages/C++/OOP Person/Book.cpp
#include<iostream>
#pragma once
using namespace std;
class Book{
    private:
        string nameBook;
        string nameAuthor;
        int numberBook;
    public:
        Book(){}
        Book(string nameBook, string nameAuthor, int numberBook)
        :nameBook(nameBook), nameAuthor(nameAuthor), numberBook(numberBook){}
        string getNameBook(){
            return nameBook;
        }
        string getNameAuthor(){
            return nameAuthor;
        }
        int getNumberBook(){
            return numberBook;
        }
        void display(){
            cout<<"Name Book: "<<getNameBook()<<endl;
            cout<<"Name Author: "<<getNameAuthor()<<endl;
            cout<<"Number Book: "<<getNumberBook()<<endl;
        }
    
};

File: Progamming Languages/C#/OOP/exercise_38/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exercise_38
{
    public class Book
    {
        public string NameBook{
            get; set;
        }

        public string NameAuthor{
            get; set;
        }

        public int NumberBook{
            get; set;
        }

        public Book(){}

        public Book(string nameBook, string nameAuthor, int numberBook)
        {
            NameBook = nameBook;
            NameAuthor = nameAuthor;
            NumberBook = numberBook;
        }

        public void display()
        {
            Console.WriteLine("Name Book: " + NameBook);
            Console.WriteLine("Name Author: " + NameAuthor);
            Console.WriteLine("Number Book: " + NumberBook);
        }

    }
}